### PR TITLE
[ACS-2158] Fix Office to Comprehend transforms via text

### PR DESCRIPTION
**Pull request by kcichonczyk**
Replaced transformer for tranforming microsoft-specific extensions to txt from LibreOffice to TikaAuto.

### CODE REVIEW

File: helm/alfresco-content-services/templates/config-transformer-routes.yaml [hunk 1]
```diff
@@ -96,7 +96,19 @@ data:
             {"sourceMediaType": "application/vnd.ms-excel",                                                   "maxSourceSizeBytes": -1, "targetMediaType": "application/vnd.alfresco.ai.features.v1+json" },
             {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",          "maxSourceSizeBytes": -1, "targetMediaType": "application/vnd.alfresco.ai.features.v1+json" },
             {"sourceMediaType": "application/vnd.ms-powerpoint",                                              "maxSourceSizeBytes": -1, "targetMediaType": "application/vnd.alfresco.ai.features.v1+json" },
```
Comment (alandavis) [inline helm/alfresco-content-services/templates/config-transformer-routes.yaml:98]: I think you need to remove the ms-powerpoint line as that is now covered by the new transform.

**Review by alandavis — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/templates/config-transformer-routes.yaml [hunk 2]
```diff
@@ -96,7 +96,19 @@ data:
...
+            {"sourceMediaType": "application/vnd.openxmlformats-officedocument.presentationml.presentation", "maxSourceSizeBytes": -1, "targetMediaType": "application/vnd.alfresco.ai.features.v1+json" }
+          ],
+          "transformOptions": [
+            "awsAiOptions"
+          ]
+        },
+        {
+          "transformerName": "msOutlookToComprehendEntityRecognitionViaText",
+          "transformerPipeline" : [
+            {"transformerName": "TikaAuto", "targetMediaType": "text/plain"},
```
Comment (alandavis) [inline helm/alfresco-content-services/templates/config-transformer-routes.yaml:108]: Is this the correct transformer?

**Review by alandavis — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/templates/config-transformer-routes.yaml (hunk 2 above)
Comment (kcichonczyk) [reply helm/alfresco-content-services/templates/config-transformer-routes.yaml]: replaced with OutlookMsg

**Review by pzhyland — APPROVED** [review]
LGTM

**Review by alandavis — COMMENTED** [review]
LGTM - please wait for a green build before merging

**Review by alandavis — CHANGES_REQUESTED** [review]
When we have a new transformer 'A' releases, these will need to be set in this PR before we merge.

**Review by alandavis — APPROVED** [review]

**Pull request merged**